components/FilterForm/index.js

```diff
@@ -50,43 +49,36 @@ export default function FilterForm({ onFilterUpdate, plants }) {
         onChange={(e) => setPlantSize(e.target.value)}
       >
         <option value="">Select Size</option>
-        <option value="Small to Medium">Small to Medium</option>
-        <option value="Medium">Medium</option>
-        <option value="Medium to Large">Medium to Large</option>
-        <option value="Large">Large</option>
+        <option value="Small">Small (15cm-50cm)</option>
+        <option value="Medium">Medium (50cm-90cm)</option>
+        <option value="Large">Large (90cm+)</option>
```

Comment:
I would recommend to not capitalize values as it is convention to use lowercase for values. This is primarily for consistency and to avoid errors due to case sensitivity. :)